conf/command.conf

```diff
@@ -0,0 +1,83 @@
+# CustomCommand section must be defined before Latency monitor to support all commands
+#
+# NOTE: only support maximum 16 custom commands
+#
+# [...] must be specify
+# (...) optional. it has default value
+#
+# CustomCommand [alias1] {   ## module alias. just a delimter
```

Comment:
The **alias1** let me confuse, we should not define multi-CustomCommand in a config file

Reply:
Okay, it will be same as before.